cmd/cofidectl/cmd/federation/federation.go

```diff
@@ -135,12 +135,7 @@ func checkFederationStatus(ctx context.Context, kubeConfig string, from *trust_z
 	}
 
 	// Bundle does not exist at all on opposite trust domain
-	_, ok := compare[to].federatedBundles[from.TrustDomain]
```

Comment:
We also need to remove the check on L144.

Reply:
Sorry, yeah. But do you mean the check on L148 to be removed, so both comparisons look like

```
	// Bundle does not exist at all on opposite trust domain
	_, ok := compare[from].federatedBundles[to.TrustDomain]
	if !ok {
		return "Unhealthy", nil
	}

	// Bundle does not match entry on opposite trust domain
	if compare[from].serverCABundle != compare[to].federatedBundles[from.TrustDomain] {
		return "Unhealthy", nil
	}
```